Add a readme

kanevk opened the pull request:
This PR improves the following things:
 - removes `keyring` and uses the config files for the persistence of the master token
 - adds a script that generates the master token and puts it in the config
 - adds minimal documentation

kanevk committed:
Simplify the login

kanevk committed:
Update the Readme

scdekov commented on README.md:
````diff
@@ -1,3 +1,47 @@
+git clone git@github.com:kanevk/gkeep-files-sync.git
+```
+
+3. Open virtual environment
````
missing `cd gkeep-files-sync` here?

scdekov commented on generate_token.py:
```diff
@@ -0,0 +1,13 @@
+import gkeepapi
+import json
```
imports ordering and 2 new lines after last import like in the previous PR.

scdekov commented on generate_token.py:
```diff
@@ -0,0 +1,13 @@
+keep.login(config['email'], config['password'])
+token = keep.getMasterToken()
+
+config['token'] = token
```
PO keep this on 1 line `config['token'] = keep.getMasterToken()`

kanevk committed:
Merge pull request #5 from kanevk/allow-syncing-custom-notes

Allow syncing custom notes

kanevk committed:
Fix incorrect sync down behaviour

The file flag "r+" is for both reading and writing, but the writing will
be executed as follows:
 - if an read have been performed the pointer for writing will be at EOF
 - writing will append text, not replace the content

 For our use case which is reading and replacing the content of the file
 the flag is "w+".
